docs/example-confluence/src/connections/apiKeyConnection.ts

```diff
@@ -0,0 +1,21 @@
+import { IConnectionSchema } from "@cognigy/extension-tools";
+
+/**
+ * This file defines a 'schema' for a connection of type 'api token' and 'email'.
+ * The connection needs to be referenced in the node that wants to
+ * use the connection:
+ * - see 'nodes/executeCognigyApiRequest.ts'
```

Comment:
There are no nodes in this example. May be something like:

```suggestion
 * The connection can be referenced in nodes and knowledge
 * - see 'knowledge.ts'
```